src/ResourceManager/AnalysisServices/Commands.AnalysisServices/help/New-AzureRmAnalysisServicesFirewallConfig.md

```diff
@@ -0,0 +1,71 @@
+﻿---
+external help file: Microsoft.Azure.Commands.AnalysisServices.dll-Help.xml
+Module Name: AzureRM.AnalysisServices
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.analysisservices/new-azurermanalysisservicesfirewallconfig
+schema: 2.0.0
+---
+
+# New-AzureRmAnalysisServicesFirewallConfig
```

Comment:
be sure to regenerate help files after you finish making changes to the cmdlets.

Reply:
sure